airflow/cli/cli_parser.py

```diff
@@ -738,6 +738,14 @@ def _check(value):
     help="Kubernetes Namespace. Default value is `[kubernetes] namespace` in configuration.",
 )
 
+ARG_MIN_PENDING_MINUTES = Arg(
+    ("--min-pending-minutes",),
+    default=30,
+    type=positive_int(allow_zero=False),
+    help="Pending pods created before the time interval are to be cleaned up, "
+         "measured in minutes. Default value is 30(m). The minimum value is 5(m).",
```

Comment:
```suggestion
    "measured in minutes. Default value is 30(m). The minimum value is 5(m).",
```

Reply:
@potiuk thanks for the reminder, added to the lastest commit